packages/nhost_sdk/lib/src/auth.dart

```diff
@@ -256,48 +258,34 @@ class Auth {
   ///
   /// Returns an [AuthResponse] with its fields unset.
   ///
-  /// Throws an [ApiException] if logout fails.
-  Future<AuthResponse> logout({
+  /// Throws an [ApiException] if sign out fails.
+  Future<AuthResponse> signOut({
     bool all = false,
   }) async {
-    log.finer('Attempting logout');
+    log.finer('Attempting sign out');
     final refreshToken =
         await _authStore.getString(refreshTokenClientStorageKey);
     try {
       await _apiClient.post(
-        '/logout',
-        query: {
-          'refresh_token': refreshToken,
-        },
-        data: {
+        '/signout',
+        jsonBody: {
+          'refreshToken': refreshToken,
           'all': all,
         },
       );
-      log.finer('Logout successful');
+      log.finer('Sign out successful');
     } catch (e, st) {
-      log.finer('Logout failed', e, st);
+      log.finer('Sign out failed', e, st);
 
       // noop
-      // TODO(shyndman): This probably shouldn't be a noop. If a logout fails,
+      // TODO(shyndman): This probably shouldn't be a noop. If a signout fails,
       // particularly in the ?all=true case, the user should know about it
     }
 
     await _clearSession();
     return AuthResponse(session: null, user: null);
   }
 
-  /// Activates a user.
```

Comment:
@shyndman are the confirmations now being handled by nhost directly? For example activation of the account, confirm email change, confirm password change. I see there is an optional redirect now. Is the intention that if not provided then nhost will manage it?

Reply:
Hey @pjindal91,

Yes, that's correct. Nhost sends out an email containing a link to an Nhost-managed confirmation endpoint. Upon confirmation, the user will be redirected to the address you've specified (which can be an app/universal link).

If not provided, the user will be redirected to an application default, specified via the `AUTH_CLIENT_URL` environment variable.

Note that it appears there are some other rules surrounding this. You can't redirect to arbitrary URLs. The logic is here: https://github.com/nhost/hasura-auth/blob/main/src/helpers.ts#L150

Reply:
Ah got it. Thanks for the explanation @shyndman